src/babel.js

```diff
@@ -40,11 +41,21 @@ export default function({ types: t }) {
           state.ignoreClosing = 0
         }
 
+        const tag = path.get('name')
+
         if (
           name &&
           name !== 'style' &&
           name !== STYLE_COMPONENT &&
-          name.charAt(0) !== name.charAt(0).toUpperCase()
+          (name.charAt(0) !== name.charAt(0).toUpperCase() ||
+            Object.values(path.scope.bindings).some(binding =>
+              binding.referencePaths.some(r => r === tag)
+            ) ||
+            (attributes &&
```

Comment:
can we remove this feature for now?

Reply:
Sure, if we want to add it at a later point let me know